artifacts/manifests/podspec.yaml

```diff
@@ -13,7 +13,6 @@ initContainers:
       cpu: 300m
       memory: 200Mi
     limits:
```

Comment:
don't remove cpu limits please. Make them configurable like memory.

Reply:
It's done.